proto/viam/app/v1/billing.proto

```diff
@@ -80,35 +94,83 @@ message GetCurrentMonthUsageSummaryResponse {
   double total_usage_without_discount = 11; // returns amt without any discounts applied
 }
 
+message GetCurrentMonthUsageRequest {
+  string org_id = 1;
+}
+
+message GetCurrentMonthUsageResponse {
+  google.protobuf.Timestamp start_date = 1;
+  google.protobuf.Timestamp end_date = 2;
+  double cloud_storage_usage_cost = 3;
+  double data_upload_usage_cost = 4;
+  double data_egres_usage_cost = 5;
+  double remote_control_usage_cost = 6;
```

Comment:
We aren't actually showing this now but I am happy to leave this in as we will show it eventually and then its hardcoded in the BE and not in the FE. Just remember in your next PR to hardcode this to 0.0 for now.

Reply:
+1. Yup, I added it because I saw it on the figma and thought it would be useful to have going forward.